src/popup/compatMode.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="compatMode.css"/>
+  </head>
+
+<body>
+  <input id="unbreakCheck" type="checkbox" name="ubreak"/>
+  <label for="unbreak">Unbreak Mode</label>
+  <div> 
+    <p>www.domain.com not working? Turn on Unbreak Mode. This might help if:</p>
+    <ul>
+      <li>Website looks broken or unusual</li>
+      <li>Forms, comments, or widgets not loading</li>
+      <li>Logging in isn’t working</li>
+      <li>Can’t complete a form or transaction</li>
+      <li>Can’t view a video</li>
+    </ul>
+  </div>
+</body>
+<script src="compatMode.js"></script>
```

Comment:
Might be worthwhile to eventually move the script tag at L23 up into the head element, below where compatMode.css is linked in and do something like `<script src="compatMode.js" defer></script>`.

Reply:
will do, thanks for the review!